debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresConnection.java

```diff
@@ -294,6 +312,18 @@ private static TypeRegistry initTypeRegistry(Connection db, Map<String, Integer>
         return typeRegistryBuilder.build();
     }
 
+    private static int getSize(final TypeInfo typeInfo, final int oid) {
+        int size = typeInfo.getPrecision(oid, TypeRegistry.NO_TYPE_MODIFIER);
+        if (size == 0) {
+            size = typeInfo.getDisplaySize(oid, TypeRegistry.NO_TYPE_MODIFIER);
```

Comment:
So what's the display size? Is it right to fall back to it?

Reply:
This is completely taken from postgres JDBC driver, so it is right fallback
https://github.com/pgjdbc/pgjdbc/blob/48b98971d085a7988b67a31cf5ff9fb52c5534e5/pgjdbc/src/main/java/org/postgresql/jdbc/PgDatabaseMetaData.java#L1515-L1518

Reply:
Ok.